docs/networking/harvester-network.md

```diff
@@ -87,6 +87,7 @@ Users can now create a new VM using the above configured VLAN network,
 
 !!! note
     - Only the first NIC will be enabled by default. Users can either choose to use a management network or a VLAN network. 
+    - You will need to be careful to configure multiple NICs VM to avoid connectivity issue. You can refer to the [knowledge base](https://harvesterhci.io/kb/multiple-nics-vm-connectivity) for more details.
```

Comment:
```suggestion
    - You need to be careful to configure virtual machines with multiple NICs to avoid connectivity issues. You can refer to the [knowledge base](https://harvesterhci.io/kb/multiple-nics-vm-connectivity) for more details.
```